### PR TITLE
Feature/emily-daniel-catsuperhero modal

### PR DESCRIPTION
**Pull request by EmilyMarschallNiswonger**
### Tracking Info

Resolves #<issue number>

Make sure your branch name conforms to: `<feature|staging|bugfix|...>/<username>/<3-4 word description separated by dashes>`. Otherwise, please rename your branch and create a new PR.

### Changes

What changes did you make?

- TODO

### Testing

How did you confirm your changes worked?

- TODO

### Confirmation of Change

Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

TODO...

### CODE REVIEW

**Review by jacobayu — DISMISSED** [review]
Hello, really nice work, it looks so good. Just a few things I noticed

1. I noticed that if you close the modal by clicking out of it (not pressing the X), to re-open the it you have to double click the cloud to open it again. You probably have to flip open again like you did on line 101.

2. The buttons for the resource links on the modal don't change color when you hover over them. 

3. Last thing, for smaller screen sizes, the buttons at the bottom get pushed out when there is a lot of text (screenshot below

 
Screen Shot 2023-04-30 at 12 35 30 PM

File: src/lib/modal.svelte [hunk 1]
```diff
@@ -0,0 +1,379 @@
...
+      }}
+    />
+    {#each entries as entry, index}
+      {#if cur == index}
+        <div class="content" in:fly={{ x: dir ? 50 : -50 }} out:fly={{ x: dir ? -50 : 50 }}>
+          <!-- svelte-ignore a11y-click-events-have-key-events -->
+          <div
+            class="x-out"
+            on:click={() => {
+              open = !open;
```
Comment (jacobayu) [inline src/lib/modal.svelte:101]: I think we have to flip open when the user clicks out of the modal as well instead of using the X

**Review by EdwardLinS — DISMISSED** [review]
Hi, I think this modal looks fantastic! However, I am encountering the same problem that Jacob is finding which are needing to click twice when the user clicks outside the modal to close it and some of the navigation buttons being pushed out on smaller screen sizes, specifically on step 5 and 6. Other than that, this look great! Nice work.

**Review by Cubified — APPROVED** [review]
LGTM 🚢 

Made some responsiveness fixes, everything else is from running the linter and merging with upstream.

**Review by jacobayu — APPROVED** [review]
still unsure if the buttons are supposed to change buttons on hover, but other than that LGTM!

**Pull request merged**